crates/matrix-sdk-crypto/src/olm/account.rs

```diff
@@ -587,6 +584,23 @@ impl ReadOnlyAccount {
         }
     }
 
+    /// Create a fresh new account, this will generate the identity key-pair.
+    #[allow(clippy::ptr_arg)]
```

Comment:
Is this `allow` really needed?

Reply:
Fixed by: 8b673ef64be3522a343aa2fb845bede7d1049d1d.